src/sentry/services/hybrid_cloud/app/service.py

```diff
@@ -82,6 +83,14 @@ def get_sentry_app_by_slug(self, *, slug: str) -> RpcSentryApp | None:
     def get_installation_by_id(self, *, id: int) -> RpcSentryAppInstallation | None:
         pass
 
+    def installation_by_id(self, *, id: int) -> RpcSentryAppInstallation | None:
+        """
+        Get a sentryapp install by id
+
+        This method is cached wrapper around get_installation_by_id()
```

Comment:
```suggestion
        This method is a cached wrapper around get_installation_by_id()
```

Reply:
Would it be worth adding `cached` in the method name to make this behavior bit more obvious?

Reply:
I'm not sure. We don't have `cached` in the other cached methods.